newIDE/app/src/UI/Dialog/index.js

```diff
@@ -105,7 +105,7 @@ export default (props: Props) => {
           fullWidth
           fullScreen={size === 'small'}
           maxWidth={maxWidth !== undefined ? maxWidth : 'md'}
-          disableBackdropClick
+          disableBackdropClick={modal !== undefined ? false : true}
```

Comment:
reconsider what happen if you:
don't pass modal,
pass `modal={true}`,
pass `modal={false}`,

false must be equivalent to not passing modal.

Reply:
modified